docs/endpoint.md

```diff
@@ -5,8 +5,103 @@ The endpoint is one of the components provided by Tangle-accelerator, running on
 The encrypted message would be sent to Tangle with a streaming message channel API. The streaming message channel API would ensure the order of messages in the channel. The user who wants to fetch/send message to Tangle needs to provide `data_id`, `key` and `protocol` to identify a specific message.
 A message sent by endpoint needs to be encrypted locally, which avoids message being peeked and modified. 
 
-# How to use
+# How to build endpoint
+
+## Setup sierra dev environment 
+The endpoint uses the sierra framework as based runtime system. Needed to setup the sierra dev environment to build endpoint as specific target.
```

Comment:
Developers need to set up the sierra dev environment to build endpoint as specific target.